src/ipfs-core/index.js

```diff
@@ -18,6 +20,7 @@ function IPFS (repo) {
     repo = defaultRepo()
   }
   const bs = new BlockService(repo)
+  const ds = new DAGService(bs)
```

Comment:
yeah, +1 for explicitly naming them `blockService` and `dagService` respectively.